filebeat/config/config.go

```diff
@@ -117,14 +118,20 @@ func (config *Config) FetchConfigs() {
 
 	if err != nil {
 		log.Fatal("Could not use config_dir of: ", configDir, err)
```

Comment:
Instead of logging the message here, we could just return the error. Can be done in a later PR.